docs/blog/2018-12-15-develop-and-deploy-your-own-react-monorepo-app/index.md

```diff
@@ -0,0 +1,598 @@
+
+```
+{
+  type: "static",
+  alias: "lerna-monorepo-blog",
+  static: {
+    public: "public"
+  },
+  public: true
+}
+```
+
+Note: `public: true` needs to be specified since you’ll most likely have a free Now account. Having a publicly facing site is one of the restrictions of the free tier. The entry file to our static site is also called `public` so don’t conflate the two.
+
+Push these changes to your repo to kick off your build pipeline. If everything was successful then your build will have passed and your site will have deployed. Go to Travis, open up your blog build so you have access to the logs. Scroll right to the bottom to where it says ‘deploying application’ and uncollapse the section. This’ll be the URL that Now deployed your site to. It should look something like `https://buildsjdoe383jd.now.sh`. Copy that url and go back to your command line. Run the following command:
+
+`now alias <your randomly generated URL> <your chosen alias>`
+I chose ‘lerna-monorepo-blog’ as my alias, so that one won’t be available for you to use, so choose your own. When you’ve run the command, there will be feedback saying that the alias was successfully created. If not, then it’s likely the command was written incorrectly or the alias is already in use by someone else. Once you’ve added an alias successfully for the blog, do the same for your shop package.
+
+Even if you push everything forward now the site’s navigation will still be broken. The very last thing we need to do is add our alias URL as an environment variables in our `.env.production` file. In your blog package’s `.env.production` go ahead and add the following:
+
+`BLOG_URL=https://<your chosen alias>.now.sh/`
+
+Do the same for your shop url, commit and push. Congrats!
+
+### Conclusion
+
+You should now be able to access the site and navigate easily between your shop and your blog. Great job! If things didn’t work out quite as planned then don’t sweat it. This tutorial is made of tiny moving parts which make for loads of opportunities for things to go wrong. I’ll try to make myself available to answer any messages left in the comments regarding any issues you come across.
+
+### Thanks for reading!
+
+Adding to your arsenal of Web Architecture tools is an excellent way of improving your capabilities as a developer or engineer. Monorepos aren’t perfect but it might just work for your next project. Good luck.
+
+If you enjoyed the article then please reach out to me on [twitter](https://twitter.com/andricokaroulla?lang=en)!
+
+### You can also check out my other articles below:
```

Comment:
Please remove this section, since these other articles are not directly related to the topic at hand.